dockerfile-image-update/src/test/java/com/salesforce/dockerfileimageupdate/subcommands/impl/ParentTest.java

```diff
@@ -122,7 +125,7 @@ public void testParentCommandSuccessfulForS3ImageStore() throws Exception {
                 "image", Constants.TAG,
                 "tag", Constants.STORE,
                 "s3://store", Constants.SKIP_PR_CREATION,
-                false);
+                false, RATE_LIMIT_PR_CREATION, "500-per-60s");
```

Comment:
This looks like old format, no?

Reply:
the correct values weren't required for the test. removed it.